src/app/pages/common/entity/table/expandable-table/expandable-table.component.ts

```diff
@@ -22,10 +25,25 @@ export class ExpandableTableComponent {
   disabled = false;
   isEmpty = true;
   isExpanded = false;
+  _tableConf: InputExpandableTableConf;
```

Comment:
1. Underscores [violate our code style.](https://github.com/airbnb/javascript#naming--leading-underscore)
2. The underlying variable is still public. It looks like you could benefit from using ngOnChanges more than a setter.